aes/decryptor_test.go

```diff
@@ -0,0 +1,108 @@
+package aes
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+	"testing/quick"
+)
+
+func TestDecrypt(t *testing.T) {
+	expected := "foo"
+	key := strings.Repeat("f", 32)
```

Comment:
fair enough. I still think you should treat this as an `[]byte`, not a printable string, because others who read the code may get the wrong idea and think this is a passphrase when it's really just bytes.